tests/TypeReconciliation/InArrayTest.php

```diff
@@ -377,24 +377,7 @@ function assertInArray($x, $y) {
 
                         return $x;
                     }',
-                'error_message' => 'RedundantConditionGivenDocblockType - src' . DIRECTORY_SEPARATOR . 'somefile.php:8:30 - Docblock-defined type int for $x is never string',
-            ],
-            'assertNegatedInArrayOfNotIntersectingTypeTriggersMixedReturnStatement' => [
```

Comment:
would become the same as assertNegatedInArrayOfNotIntersectingTypeTriggersRedundantCondition L317